src/main/java/com/marklogic/client/ext/datamovement/consumer/WriteRowToTableauConsumer.java

```diff
@@ -0,0 +1,250 @@
+package com.marklogic.client.ext.datamovement.consumer;
+
+import com.marklogic.client.MarkLogicIOException;
+import com.marklogic.client.example.cookbook.datamovement.TypedRow;
+import com.marklogic.client.type.*;
+import com.tableausoftware.TableauException;
```

Comment:
Where do you get the tableau jar from? I couldn't find it in jcenter or mavenCentral. I'm wondering if they have a license that allows for it to be added to this project. Otherwise, we have a bit of a problem with how to compile this code. 

If the jar can be depended on or added to this project, we'd want to make it a "compileOnly" Gradle dependency, which means it's used to compile the code but is not a runtime dependency.